app/ui-message/client/actionButtons/messageBox.ts

```diff
@@ -0,0 +1,25 @@
+import { Session } from 'meteor/session';
+import { IUIActionButton } from '@rocket.chat/apps-engine/definition/ui';
+
+import { Rooms } from '../../../models/client';
+import { messageBox } from '../../../ui-utils/client';
+import { applyButtonFilters } from './lib/applyButtonFilters';
+
+const getIdForActionButton = ({ appId, actionId }: IUIActionButton): string => `${ appId }/${ actionId }`;
+
+const APP_GROUP = 'Create_new';
+
+// eslint-disable-next-line no-void
+export const onAdded = (button: IUIActionButton): void => void messageBox.actions.add(APP_GROUP, button.nameI18n, {
+	id: getIdForActionButton(button),
+	icon: 'arrow-back',
```

Comment:
The icon name expected in this property is not from the same source as the `icon` [in the tabbar/toolbox](https://github.com/RocketChat/Rocket.Chat/blob/feat/add-ui-buttons-to-contexts/app/ui-message/client/actionButtons/tabbar.ts#L11), but it's rather from a much smaller set of icons. Can we unify this? If not, how to better explain that to apps developers?

Reply:
Why not? I don't get it.

Reply:
Because the Toolbox is rendered [via Fuselage](https://github.com/RocketChat/Rocket.Chat/blob/7c9658f2257bcf9ec84d8639e73fa2b88cffc0dd/client/components/Header/ToolBoxAction.tsx#L4) and the message box is rendered [with Blaze](https://github.com/RocketChat/Rocket.Chat/blob/a0249005961091c06b820d807d42e36f75be4996/app/ui-message/client/messageBox/messageBox.html#L41)

Reply:
Ah, I see. `@rocket.chat/icons` can be used independently of Fuselage; I'll review which ones can be safely replaced.